authors/apps/profiles/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.urls import path
 from .views import ProfileRetrieveUpdateView, ProfileFollowAPIView, \
     ProfileFollowersAPIView, ProfileFollowingAPIView
+from .views import (
```

Comment:
Any reason why you have two imports from `.view`, also you are importing `ProfileRetrieveUpdateView` twice. Kindly look into it

Reply:
worked on.